Source/WebCore/rendering/style/StyleGradientImage.cpp

```diff
@@ -1279,51 +1427,45 @@ Ref<Gradient> StyleGradientImage::createGradient(const RadialData& radial, const
 
 // MARK: - Prefixed Radial create.
 
-Ref<Gradient> StyleGradientImage::createGradient(const PrefixedRadialData& radial, const RenderElement& renderer, const FloatSize& size, const RenderStyle& style) const
+Ref<Gradient> StyleGradientImage::createGradient(const PrefixedRadialData& radial, const RenderElement&, const FloatSize& size, const RenderStyle& style) const
```

Comment:
Can we eliminate the `renderer` argument entirely instead of passing it and ignoring it?